apiary.apib

```diff
@@ -954,11 +954,570 @@ Outputs the list of peers that were removed.
 
 # Group config
 
-## edit
+## config [POST /config{?arg1,arg2}{&bool,json}]
 
-## replace
+'ipfs config' controls configuration variables. It works
+much like 'git config'. The configuration values are stored in a config
+file inside your IPFS repository.
 
-## show
++ Parameters
+    + arg1: "Datastore.Path" (string, required) - The key of the config entry.
+    + arg2: "~/.ipfs/datastore" (string, required) - The value to set the config entry to.
```

Comment:
Isn't this optional? See https://github.com/ipfs/http-api-spec/blob/feature/config/apiary.apib#L1038